sdk/executor/src/main/java/com/mesosphere/sdk/executor/ProcessTask.java

```diff
@@ -251,4 +258,61 @@ private boolean waitExit(final Duration timeout) {
         }
     }
 
+    /**
+     * Sets up the config files on the executor side.
+     *
+     * @param taskInfo The {@link TaskInfo} to extract the config file data from
+     * @throws IOException if the data in the taskInfo is not valid or the config can't be written to disk
+     */
+    private static void setupConfigFiles(TaskInfo taskInfo) throws IOException {
```

Comment:
Generally don't like adding code to the CustomExecutor as I'm trying to get rid of it.  But we can rip this code out later for a standalone tool.  Taking it from here or a Utils class makes no difference in the end really.